logic/src/commonMain/kotlin/com/wire/kalium/logic/data/message/MessageContent.kt

```diff
@@ -279,6 +281,7 @@ fun MessageContent?.getType() = when (this) {
     is MessageContent.MemberChange.CreationAdded -> "MemberChange.CreationAdded"
     is MessageContent.MemberChange.FailedToAdd -> "MemberChange.FailedToAdd"
     is MessageContent.MLSWrongEpochWarning -> "MLSWrongEpochWarning"
+    is MessageContent.VerificationDegraded -> "Verification status degraded"
```

Comment:
what do you think? of having it like:```suggestion
    is MessageContent.VerificationDegraded -> "ConversationVerification.Degraded"
```